aws-replicator/aws_replicator/client/auth_proxy.py

```diff
@@ -47,6 +50,9 @@
 CONTAINER_CONFIG_FILE = "/tmp/ls.aws.proxy.yml"
 CONTAINER_LOG_FILE = "/tmp/ls-aws-proxy.log"
 
+# default bind host if `bind_host` is not specified for the proxy
+DEFAULT_BIND_HOST = "127.0.0.1"
```

Comment:
maybe this extension may benefit from also using `localstack_host`

https://github.com/localstack/localstack-extensions/pull/39/files#diff-d34673c9767f9d401b3a924ff4c9a9ca9bb80c8ee9b79a1b114e34abaeddaacf